pyam/utils.py

```diff
@@ -181,13 +176,9 @@ def read_file(path, *args, **kwargs):
     return format_data(read_pandas(path, *args, **kwargs), **format_kwargs)
 
 
-def format_data(df, index, **kwargs):
-    """Convert a pandas.Dataframe or pandas.Series to the required format"""
-    if isinstance(df, pd.Series):
-        df.name = df.name or "value"
-        df = df.to_frame()
+def _convert_r_columns(df):
+    """Check and convert R-style year columns"""
 
-    # check for R-style year columns, converting where necessary
     def convert_r_columns(c):
```

Comment:
Repeating the function name within is a bit confusing, propose to be more explicit.
```suggestion
    def strip_R_integer_prefix(c):
```